modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -811,6 +812,28 @@ func (suite *KeeperTestSuite) TestPruneAcknowledgements() {
 			},
 			nil,
 		},
+		{
+			"success: limit wraps around due to uint64 overflow",
+			func() {
+				// Send 10 packets from B -> A, creating 10 packet receipts and 10 packet acks on A.
+				suite.sendMockPackets(path, 10, true)
+			},
+			func() {
+				limit = math.MaxUint64
+			},
+			func(pruned, left uint64) {
+				// Nothing should be pruned, by passing in a limit of math.MaxUint64, overflow occurs
+				// when initializing end to pruningSequenceStart + limit. This results in end always being
+				// equal to start - 1 and thereby not entering the for loop.
+				// We expect 10 acks, 10 receipts and pruningSequenceStart == 1 (loop not entered).
```

Comment:
crazy, so it doesn't panic, it just wraps around. I guess by construction this is safe. Should we add an in-line comment in the code in case someone tries to refactor it?